content/library/api/connections/connections.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Connections and databases
+slug: /library/api-reference/connections
+---
+
+# Connections and databases
+
+## Setup your connection
+
+<TileContainer>
+<RefCard href="/library/api-reference/connections/st.experimental_connection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/experimental-connection.png" />
+
+#### Create a connection
+
+Connect to a database and return a connection object.
+
+```python
+st.experimental_connection('pet_db', type='sql')
+```
+
+</RefCard>
+</TileContainer>
+
+## Built-in connections
+
+<TileContainer>
+
+<RefCard href="/library/api-reference/connections/st.connections.sqlconnection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/sqlalchemy.png" />
```

Comment:
I would suggest a different image, maybe one of these:

https://uxwing.com/database-db-icon/

https://commons.wikimedia.org/wiki/File:Database-icon.svg

Reply:
Added the images on db061a1. Btw, I also added an image for `st.experimental_connection`.  LMK if you think it works or if I should remove it like you initially suggested:

![image](https://user-images.githubusercontent.com/20672874/234293651-28b81efd-9682-473a-a326-f585eff9c543.png)

Reply:
I like this image!!! Nice!! 🎉